docs/UserGuide.md

```diff
@@ -215,32 +205,61 @@ absolute filePath.
 Format: `import i/FILEPATH`
 - imports the persons saved in `FILEPATH` to `avengersassemble.json`
 
+You can refer to the [command format summary](#command-format-summary) for more guidelines for each parameter.
+
 ### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
+### Archiving data files `[coming in v2.0]`
+
+_Details coming soon ..._
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Additional Information
+
 ### Saving the data
 
-AA data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+All data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AA data are saved automatically as a JSON file `[JAR file location]/data/avengersassemble.json`. Advanced users are welcome to update data directly by editing that data file.
+All data are saved automatically as a JSON file, `[JAR file location]/data/avengersassemble.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <box type="warning" seamless>
 
 **Caution:**
-If your changes to the data file makes its format invalid, AA will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
-Furthermore, certain edits can cause AA to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+If your changes to the data file makes its format invalid, Avengers Assemble will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br/>
+Furthermore, certain edits can cause the Avengers Assemble application to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </box>
 
-###
+### Command Format Summary
 
-### Archiving data files `[coming in v2.0]`
+Some commands require you to include parameters. These parameters are identified by prefixes. Here are a list of valid prefixes and what they each refer to.
 
-_Details coming soon ..._
+<box type="info" seamless>
```

Comment:
Nice addition! The overly verbose table has been condensed and referenced which makes it much more convenient.